release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -403,6 +403,13 @@ The following updates have been made to the status management of `EgressFirewall
 
 With this update, MetalLB exposes additional metrics relating to communication between MetalLB and Border Gateway Protocol (BGP) peers. For more information, see xref:../networking/metallb/metallb-troubleshoot-support.adoc#nw-metallb-metrics_metallb-troubleshoot-support[MetalLB metrics for BGP and BFD].
 
+[id="ocp-4-15-all-multicast"]
+==== Supporting all-multicast mode
+
+{product-title}release now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```

Comment:
Is there a reason to have release here? Or maybe you're looking to add the version attribute? {product-version}

```suggestion
{product-title} now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```